src/lib/validators.js

```diff
@@ -100,6 +100,23 @@ const validators = {
     if (dot.pick(sub, data) === undefined && value === undefined) {
       errors.push({ type, sub, key, value, message: `Value required because '${sub}' is undefined`})
     }
+  },
+
+  /**
+   * Validator equalTo method, used by model
+   * @param {Object} def The property configuration
+   * @param {*} value The value being validated
+   * @param {string} key The key name of the property
+   * @param {Array<{type: string, sub: string|number, key: string, value: *, message: string}>} errors An error array
+   * to which any additional error objects will be added
+   * @param {Object} data The full initial data object
+   */
+  equalTo: function(def, value, key, errors, data) {
+    const type = 'equalTo'
+    const sub = def.equalTo
+    if (dot.pick(sub, data) !== value) {
+      errors.push({ type, sub, key, value, message: `Value doesn\'t match '${sub}' value`})
```

Comment:
Can you change this to the instructional `Value must match ${sub} value ${dot.pick(sub, data)}`